.aws/src/event-rules/prospect-events/prospectEventRules.ts

```diff
@@ -39,33 +50,82 @@ export class ProspectEvents extends Resource {
     super(scope, name);
 
     // pre-existing queues (prod and dev) created by prospect-api
-    this.sqs = new sqs.DataAwsSqsQueue(this, `prospect-${config.environment}-queue`, {
-      name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue`
-    })
+    this.sqs = this.createSqsForProspectEvents(
+      this.sqsIdForProspectGenerationEvent,
+      this.sqsNameForProspectGenerationEvent
+    );
 
-    this.sqsDlq = new sqs.DataAwsSqsQueue(this, `prospect-${config.environment}-dlq`, {
-      name: `ProspectAPI-${config.environment}-Sqs-Translation-Queue-Deadletter`
-    });
+    // create a dlq for all Prospect events
+    this.sqsDlq = this.createSqsForProspectEvents(
+      this.dlqIdForProspectEvents,
+      this.dlqNameForProspectEvents
+    );
+
+    // create an SNS topic for all Prospect events except for prospect-generation
+    this.snsTopic = this.createSnsForProspectEvents();
 
-    this.createProspectEventRules();
+    this.createProspectGenerationEventRule();
     this.createPolicyForEventBridgeToSqs();
 
+    // setting up prospect-dismiss event rule
+    this.createProspectEventRule(
+      'Dismiss-Prospect-Events-Rule',
+      eventConfig.prospectDismiss.source,
+      eventConfig.prospectDismiss.detailType
+    );
```

Comment:
Should we rename this event to `Prospect-Dimiss-Events-Rule` (being set below in the function on **line 184**) ? 

This is not consistent with the other event rule name on **line 144** below

Reply:
i like the `{Entity}-{Action}-Events-Rule` pattern, and would vote to change the name on line 144 if it won't cause any issues.